src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -1096,4 +1096,20 @@ public void testNoReadsReblockedOutputAsNoCall() {
         Assert.assertFalse(g.hasAD());
         Assert.assertFalse(g.hasPL());
     }
+
+    @Test
+    public void testMixHaploidDiploidHighAltSite() {
+        final File output = createTempFile("MixHaploidDiploidHighAltSite", ".vcf");
+        final ArgumentsBuilder args = new ArgumentsBuilder();
+        args.addReference(hg38Reference)
+                .addVCF("gendb://" + toolsTestDir + "/walkers/GenotypeGVCFs/mixHaploidDiploidHighAlt")
```

Comment:
I think I'd rather the test you added start from VCFs than commit a weird genomicsDB structure to the repo that might break if we update GenomicsDB (which admittedly doesn't happen that often).  Also the VCF is human-readable, whereas I have no idea what data you just added here. Take a look at testMaxAltsToCombineInGenomicsDB:
        final File tempGenomicsDB2 = GenomicsDBTestUtils.createTempGenomicsDB(inputs, interval);
        final String genomicsDBUri2 = GenomicsDBTestUtils.makeGenomicsDBUri(tempGenomicsDB2);